Framework/DataHandling/src/LoadILLIndirect.cpp

```diff
@@ -209,7 +210,7 @@ void LoadILLIndirect::initWorkSpace(
   m_localWorkspace = WorkspaceFactory::Instance().create(
       "Workspace2D",
       m_numberOfHistograms + monitorsData.size() + m_numberOfSimpleDetectors,
-      m_numberOfChannels + 1, m_numberOfChannels);
+      m_numberOfChannels, m_numberOfChannels);
```

Comment:
If you have bin edges in a workspace the first of these arguments should have the `+1`.